examples/aws_ebs/README.md

```diff
@@ -0,0 +1,17 @@
+This is a simple web server pod which serves HTML from an AWS EBS
+volume.
+
+Create a volume in the same region as your node add your volume
+information in the pod description file aws-ebs-web.yaml then create
+the pod:
+
+  $ kubectl create -f aws-ebs-web.yaml
+
+Add some data to the volume if is empty:
+
+  $ echo  "Hello World" >& /var/lib/kubelet/plugins/kubernetes.io/aws-ebs/mounts/aws/{Region}/{Volume ID}/index.html
+
+You should now be able to query your web server:
+
+  $ curl <Pod IP address>
+  $ Hello World
```

Comment:
We require analytic links at the bottom of our READMEs. Shippable is currently failing this verification on your PR. Running hack/run-gendocs.sh should do that.